src/components/ciPipeline/SourceMaterials.tsx

```diff
@@ -164,16 +175,47 @@ export const SourceMaterials: React.FC<SourceMaterialsProps> = function (props)
                                         }}
                                         menuPortalTarget={document.getElementById('visible-modal')}
                                     />
+                                    <div className="h-18"></div>
```

Comment:
why are we adding this empty div

Reply:
In case of NO error , below input page was jumping.  To reduce that added height in case of no error